src/Exercise05AI.ipynb

```diff
@@ -69,21 +69,21 @@
     "\n",
     "Die folgende Implementierung des Minimax-Algorithmus ist zu großen Teilen mit der von `Exercise02AI` identisch. Als Neuerung wurde Alpha-Beta-Pruning hinzugefügt. Im Folgenden werden nur die Änderungen betrachtet:\n",
     "\n",
-    "1. Die `minimax`-Funktion bekommt zwei zusätzliche Parameter $alpha$ und $beta$ welche mithilfe von Standardwerten beim ersten Aufruf auf das positive bzw. negative Limit gesetzt werden.\n",
-    "2. Wenn der weiße Spieler am Zug ist (Maximierung) wird `maxEvaluation` auf den Wert $alpha$ gesetzt. Wie bisher wird nun jeder Zug evaluiert. Hierbei wird das Maximum der bisherigen Evaluationen jeweils als neue untere Grenze $alpha$ eingesetzt. Falls eine Evaluierung größer oder gleich $beta$ ist, wird die Suche abgebrochen (pruning) und diese Evaluierung zurückgegeben.\n",
-    "2. Falls der schwarze Spieler am Zug ist (Minimierung) wird `minEvaluation` auf den Wert $beta$ gesetzt. Wie bisher wird nun jeder Zug evaluiert. Hierbei wird das Minimum der bisherigen Evaluationen jeweils als neue obere Grenze $beta$ eingesetzt. Falls eine Evaluierung kleiner oder gleich $alpha$ ist, wird die Suche abgebrochen (pruning) und diese Evaluierung zurückgegeben.\n",
+    "1. Die `minimax`-Funktion bekommt zwei zusätzliche Parameter $\\texttt{alpha}$ und $\\texttt{beta}$ welche mithilfe von Standardwerten beim ersten Aufruf auf das positive bzw. negative Limit gesetzt werden.\n",
```

Comment:
Hm, wir haben jetzt eine bisschen inkonsequente Verwendung von \texttt und \textrm. Ich habe letzteres immer für alle Variablen genommen. Wie sollen wir das in Zukunft machen? Willst du das auch dahingehend anpassen?

Reply:
Ja, das stimmt. Ich hatte alles entsprechend zu `\textrm{}` geändert. Mit genau dieser einzigen Ausnahme. Das liegt daran, dass die Fallunterscheidung sonst keinen Unterschied zwischen den eigentlichen Variablen und dem Text zulässt. Falls du eine andere Idee hast, passen wir das gern an.

Reply:
Ok, sollen wir dann einheitlich bei \textrm für Variablen bleiben und normalen Text im Math-Mode dann als \texttt schreiben?

Reply:
Klingt vernünftig :+1:

Reply:
Fixed in 52fe017.